homeassistant/components/goodwe/sensor.py

```diff
@@ -109,6 +114,18 @@ class GoodweSensorEntityDescription(SensorEntityDescription):
         value=lambda prev, val: prev if not val else val,
         available=lambda entity: entity.coordinator.data is not None,
     ),
+    "VA": GoodweSensorEntityDescription(
```

Comment:
Please don't mix fixes and feature additions. Move the new features to a separate PR.

Reply:
Done, extracted to separate PR #87940